src/shipit_static_analysis/shipit_static_analysis/__init__.py

```diff
@@ -23,6 +25,20 @@ class Issue(object):
     - line: Line where the issue begins
     - nb_lines: Number of lines affected by the issue
     '''
+    def __init__(self):
```

Comment:
Nit: I would move this code to the `__hash__` function, to keep all things related with hashing in the same function.